.github/workflows/main.yml

```diff
@@ -21,16 +21,18 @@ jobs:
     name: Python ${{ matrix.python-version }} on ${{ matrix.os }} with numpy ${{ matrix.numpy_ver }}
     runs-on: ${{ matrix.os }}
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v2
+      uses: actions/setup-python@v4
       with:
         python-version: ${{ matrix.python-version }}
 
     - name: Install NEP29 dependencies
       if: ${{ matrix.numpy_ver != 'latest'}}
       run: |
         pip install --no-binary :numpy: numpy==${{ matrix.numpy_ver }}
+        # Need to force install of pandas compliant with NEP29
```

Comment:
I thought we just added support for pandas 1.5....

Reply:
We support pandas 1.5, but pandas 1.5 does not support numpy 1.20.  It's a problem created by pandas not supporting the NEP29 schedule that goes away Jan 31.